pkg/apply/applier.go

```diff
@@ -188,6 +199,10 @@ func (a *Applier) Run(ctx context.Context, invInfo inventory.InventoryInfo, obje
 
 		// Build list of prune validation filters.
 		pruneFilters := []filter.ValidationFilter{
+			filter.InvalidObjectFilter{
```

Comment:
Does the order of these filters matter?

Reply:
They get executed in this order. Filtering invalid objects first prevents the other filters from erroring due to an invalid object.